lib/pg.ddl.js

```diff
@@ -312,7 +318,9 @@ const generateTable = (
     uniqueSQL = '\n' + uniqueSQL + '\n';
   }
 
+  const comment = `Category: ${name}`;
   const sql =
+    createComment(comment) +
```

Comment:
I think there is no need to create a variable for `comment`, you can just write:
```js
    createComment(`Category: ${name}`);
```